lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -44,7 +44,8 @@
       - Type of database object to set privileges on.
```

Comment:
Add a note here that 'foreign_data_wrapper' and 'foreign_server' are available from '2.6' version and onwards.